### PR TITLE
RA - Add new Bridges and make them use proper HitShapes

### PR DESCRIPTION
**Pull request by MustaphaTR**
Revived #14499. This one only adds NW-SE diagonal ones, as others were buggy. But this time includes one for desert tileset too.

### CODE REVIEW

File: mods/ra/weapons/missiles.yaml [hunk 1]
```diff
@@ -246,7 +246,6 @@ TorpTube:
 			Wood: 75
 			Light: 75
 			Heavy: 100
-			Concrete: 500
```
Comment (Mailaender) [inline mods/ra/weapons/missiles.yaml]: This looks like an unrelated change?

Comment (MustaphaTR) [reply mods/ra/weapons/missiles.yaml]: No. Bridges has Concrete armor and if i don't remove that subs destroy bridges too fast due to the fact that they now have proper HitShapes. This reduces it to be more similar to what it was vs what it is now.

**Comment by MustaphaTR** [comment]
Updated, seems like i forgot to push changes to world.yaml somehow.

File: mods/ra/weapons/missiles.yaml (hunk 1 above)
Comment (Mailaender) [reply mods/ra/weapons/missiles.yaml]: Why don't they show their hitshape when I enable the debug option?

**Comment by Mailaender** [comment]
It looks like the bridge hut location needs to move as the destroyed ones don't let engineers enter:

openra-2018-09-13t130145097z

File: mods/ra/weapons/missiles.yaml (hunk 1 above)
Comment (MustaphaTR) [reply mods/ra/weapons/missiles.yaml]: Because they don't have CombatDebugOverlay trait. Should i add it with this PR, i used it for testing a lot.

Comment (abcdefg30) [reply mods/ra/weapons/missiles.yaml]: Sounds like a good idea. :+1:

**Comment by netnazgul** [comment]
Looks like sometimes adjacent tile doesn't change to damaged like it is supposed to
image
image

Also top edge diagonal is crossable between the beach and the bridge, but this is the problem for sw-ne bridge as well.

Otherwise fine for me, will do non-temperate tileset tests later also.

**Review by netnazgul — CHANGES_REQUESTED** [review]
Looks like sometimes adjacent tile doesn't change to damaged like it is supposed to

**Comment by pchote** [comment]
@MustaphaTR are you planning to revisit this? If not, can you please close this?

**Comment by pchote** [comment]
Moving to the future milestone.
